homeassistant/components/sensor/zoneminder.py

```diff
@@ -104,13 +124,22 @@ def state(self):
 
     def update(self):
         """Update the sensor."""
-        archived_filter = '/Archived:0'
+        date_filter = '1%%20%s' % self._type
+        if self._type == 'all':
+            # The consoleEvents API uses DATE_SUB, so give it
+            # something large
+            date_filter = '100%20year'
+
+        archived_filter = '/Archived=:0'
         if self._include_archived:
             archived_filter = ''
 
         event = zoneminder.get_state(
-            'api/events/index/MonitorId:%i%s.json' % (self._monitor_id,
-                                                      archived_filter)
+            'api/events/consoleEvents/%s%s.json' % (date_filter,
```

Comment:
Same as above.

Reply:
The name for the default 'all' events sensor is "<monitor name> Events", which is the same as it was previously. So it should not be a breaking change.